.github/workflows/connect-dev-release.yml

```diff
@@ -0,0 +1,105 @@
+# TODO: add tests and call it [Build/Test]
+name: "[Build] connect"
+
+permissions:
+  id-token: write # for fetching the OIDC token
+  contents: read # for actions/checkout
+
+on:
+  push:
+    paths:
+      - "packages/blockchain-link"
+      - "packages/connect/**"
+      - "packages/connect-common/**"
+      - "packages/connect-iframe/**"
+      - "packages/connect-web/**"
+      - "packages/transport"
+      - "packages/utxo-lib"
+      - "packages/utils"
+      - "submodules/trezor-common"
+      - "yarn.lock"
+
```

Comment:
If I understand well based on this configuration this branch should be running this action everytime I push to it but it is not doing it.
Does it have to be merged in develop before so it runs? @vdovhanych

Reply:
im not sure bu i would just keep it on pull_request as that will run it on every push for that pull request there is no need to do this

Reply:
```yaml
on:
  pull_request:
    paths:
      - "packages/blockchain-link"
      - "packages/connect/**"
      - "packages/connect-common/**"
      - "packages/connect-iframe/**"
      - "packages/connect-web/**"
      - "packages/transport"
      - "packages/utxo-lib"
      - "packages/utils"
      - "submodules/trezor-common"
      - "yarn.lock"
```

Reply:
this looks better, lets try

Reply:
Shouldn't we have something like below? So we run it also in develop?

```yaml
on:
  push:
    branches:
      - develop
    paths:
      - "packages/blockchain-link"
      - "packages/connect/**"
      - "packages/connect-common/**"
      - "packages/connect-iframe/**"
      - "packages/connect-web/**"
      - "packages/transport"
      - "packages/utxo-lib"
      - "packages/utils"
      - "submodules/trezor-common"
      - "yarn.lock"
  pull_request:
    paths:
      - "packages/blockchain-link"
      - "packages/connect/**"
      - "packages/connect-common/**"
      - "packages/connect-iframe/**"
      - "packages/connect-web/**"
      - "packages/transport"
      - "packages/utxo-lib"
      - "packages/utils"
      - "submodules/trezor-common"
      - "yarn.lock"
  ```

By the way I changed it to have on `pull_request:` but it is still not running here. I also created new PR with same code https://github.com/trezor/trezor-suite/pull/11227 and it still do not run there. No sure why yet.

Reply:
do we need to run it on develop as it was already run basically on develop and plus what its adding?